### PR TITLE
refactor(playground): integrate playground with client js sdk

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

Integrate playground with client js sdk

- Create a logtoClient instance and put is in as a context
- replace original login/logout logic using logtoClient api
- add `jest-setup` file to bypass the `jose` jest env issue
- As `token_set` storage is hidden to the client, display IDToken claims instead of original token_set details  after success login

## Testing
Test locally successfully sign-in and out

image

@wangsijie @xiaoyijun cc: @IceHe @gao-sun

### CODE REVIEW

**Review by xiaoyijun — APPROVED** [review]
LGTM

File: packages/playground/src/consts/url.ts [hunk 1]
```diff
@@ -1,2 +1,5 @@
 export const baseUrl = `http://${String(process.env.HOST)}:${String(process.env.PORT)}`;
-export const oidcUrl = process.env.RAZZLE_OIDC_BASE_URL ?? 'N/A';
+export const oidcDomain = process.env.RAZZLE_OIDC_BASE_DOMAIN ?? 'N/A';
+export const oidcUrl = process.env.RAZZLE_OIDC_BASE_DOMAIN
```
Comment (wangsijie) [inline packages/playground/src/consts/url.ts:3]: is this still useful?

**Review by wangsijie — APPROVED** [review]
LGTM

File: packages/playground/src/consts/url.ts (hunk 1 above)
Comment (simeng-li) [reply packages/playground/src/consts/url.ts]: removed thx

**Pull request merged**